esy-build-package/Config.re

```diff
@@ -60,7 +59,7 @@ let make = (~storePath, ~projectPath, ~localStorePath, ()) => {
         );
       };
     };
-  let%bind () = initStore(localStorePath);
+  // let%bind () = initStore(localStorePath);
```

Comment:
This seems unneeded since we're calling `initStore` via `configureStorePath` already